code/modules/antagonists/eldritch_cult/knowledge.dm/ash_lore.dm

```diff
@@ -0,0 +1,202 @@
+	cost = 1
+	route = "Ash"
+
+/datum/eldritch_knowledge/spell/ashen_shift
+	name = "Ashen Shift"
+	gain_text = "Ash is all the same, how can one man master it all?"
+	desc = "Short range jaunt that can help you escape from bad situations."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash
+	next_knowledge = list(/datum/eldritch_knowledge/ash_mark,/datum/eldritch_knowledge/essence,/datum/eldritch_knowledge/ashen_eyes)
+	route = "Ash"
+
+/datum/eldritch_knowledge/ashen_grasp
+	name = "Ashen Grasp"
+	gain_text = "Gates have opened, minds have flooded, I remain."
+	desc = "Empowers your mansus grasp to throw away enemies."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/ashen_shift)
+	route = "Ash"
+
+/datum/eldritch_knowledge/ashen_grasp/mansus_grasp_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!iscarbon(target))
+		return
+
+	var/mob/living/carbon/C = target
+	var/datum/status_effect/eldritch/E = C.has_status_effect(/datum/status_effect/eldritch/rust) || C.has_status_effect(/datum/status_effect/eldritch/ash) || C.has_status_effect(/datum/status_effect/eldritch/flesh)
+	if(E)
+		E.on_effect()
+		for(var/X in user.mind.spell_list)
+			if(!istype(X,/obj/effect/proc_holder/spell/targeted/touch/mansus_grasp))
+				continue
+			var/obj/effect/proc_holder/spell/targeted/touch/mansus_grasp/MG = X
+			MG.charge_counter = min(round(MG.charge_counter + MG.charge_max * 0.75),MG.charge_max) // refunds 75% of charge.
+	var/atom/throw_target = get_edge_target_turf(C, user.dir)
+	if(!C.anchored)
+		C.throw_at(throw_target, rand(4,8), 14, user)
+	return
+
+/datum/eldritch_knowledge/ashen_eyes
+	name = "Ashen Eyes"
+	gain_text = "Piercing eyes may guide me through the mundane."
+	desc = "Allows you to craft thermal vision amulet by transmutating eyes with a glass shard."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/ashen_shift,/datum/eldritch_knowledge/flesh_ghoul)
+	required_atoms = list(/obj/item/organ/eyes,/obj/item/shard)
+	result_atoms = list(/obj/item/clothing/neck/eldritch_amulet)
+
+/datum/eldritch_knowledge/ash_mark
+	name = "Mark of ash"
+	gain_text = "Spread the famine."
+	desc = "Your sickly blade now applies ash mark on hit. Use your mansus grasp to proc the mark. Mark of Ash causes stamina damage, and fire loss, and spreads to a nearby carbon. Damage scales with how many times the mark has spread."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/curse/blindness)
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/flesh_mark)
+	route = "Ash"
+
+/datum/eldritch_knowledge/ash_mark/eldritch_blade_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(istype(target,/mob/living))
+		var/mob/living/L = target
+		L.apply_status_effect(/datum/status_effect/eldritch/ash,1)
+
+/datum/eldritch_knowledge/curse/blindness
+	name = "Curse of blindness"
```

Comment:
suggestion: add another curse spell that mutes and emotemutes its victim

Reply:
i considered adding it, but it defeats the point of actually being stealthy at your killing ,since you can unga bunga, mute seomeone for 5 minutes and kill them easily

Reply:
I mean, your victim can still seek the safety of others once they've realized that they've been muted

maybe just make it a normal mute (with no emotemuting) then.

Reply:
while we're on the subject of voodoo, it'd be nice if you could make, like, a DNA scraper or something to extract DNA from airlocks and such. I've played around with (in-game) voodoo dolls before, and it's actually really hard to get an object that someone has touched with their bare hands (as opposed to, say, a door)

Reply:
this doesnt require an item, any atom will do

Reply:
including airlocks,tables etc.